examples/basics/scene/mesh_shading.py

```diff
@@ -24,10 +24,12 @@
 mesh = Mesh(vertices, faces, color=(.5, .7, .5, 1))
 view.add(mesh)
 
-shading_filter = ShadingFilter(shininess=args.shininess)
-mesh.attach(shading_filter)
 wireframe_filter = WireframeFilter(width=args.wireframe_width)
+shading_filter = ShadingFilter(shininess=args.shininess)
+# Note: The wireframe filter is attached before the shading filter otherwise
+# the wireframe is not shaded.
```

Comment:
Would you mind adding this information to both the filter classes docstrings?

Reply:
Sure. I started documenting.